model/system_metrics_test.go

```diff
@@ -81,6 +81,80 @@ func TestSystemMetricsFind(t *testing.T) {
 	})
 }
 
+func TestSystemMetricsFindByTaskID(t *testing.T) {
+	env := cedar.GetEnvironment()
+	db := env.GetDB()
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+	defer func() {
```

Comment:
Should this also clear the collection at the start of the test to ensure it has no data to begin with?